File: programmers/notComplete.py
def solution(participant, completion):
    answer = ''

    map = {}
    for p in participant:
        if p not in map:
            map[p] = 1
        else:
            map[p] += 1

    for c in completion:
        map[c] -= 1

    for key in map:
        if(map[key]!=0):
            answer = key
            break

    return answer

solution(['mislav', 'stanko', 'mislav', 'ana','mislav'], ['stanko', 'ana', 'mislav'])

File: programmers/javascript/notComplete.js
function solution(participant, completion) {
    let answer = '';

    let obj = {};
    for (let part of participant) {
        if (part in obj) {
            obj[part] += 1;
        } else {
            obj[part] = 1;
        }
    }

    for (let comp of completion) {
        if (comp in obj) {
            obj[comp] -= 1;
            if (obj[comp] == 0) {
                delete obj[comp];
            }
        } else {
            console.log('error');
        }
    }

    answer = Object.keys(obj)[0];

    return answer;
}

console.log(solution(["leo", "kiki", "eden"], ['eden', 'kiki']))